openshift-install-powervs

```diff
@@ -1198,6 +1203,49 @@ function setup {
   success "setup command completed!"
 }
 
+#-------------------------------------------------------------------------
+# Read the given argument from var
+#-------------------------------------------------------------------------
+function process_var {
+  var_name=$(echo "$var" | cut -d '=' -f 1)
```

Comment:
`debug_switch` should be before this as well.. as you are doing an echo.

Reply:
Resolved